tipg/dbmodel.py

```diff
@@ -164,6 +167,7 @@ class Collection(BaseModel):
     geometry_column: Optional[Column]
     datetime_column: Optional[Column]
     parameters: List[Parameter] = []
+    last_catalog_update: datetime = datetime.now()
```

Comment:
default should be set on the post validation step! the issue here is that you'll create a default when we import the dbmodel submodule